writing-content/image.mdx

```diff
@@ -32,6 +32,13 @@ Although not required, we recommend adding the `alt` and `title` attributes to i
 
 </Tip>
 
+<Info>
+
+Currently, we support images up to 5MB. If you have larger images, we recommend hosting the images yourself using services like Cloudinary or S3, and adding the link will render the image in your doc. 
```

Comment:
you can change this to "like Cloudinary or S3. You can then use the hosted URL in an `<img>` tag"